### PR TITLE
[CP] Open CP to all Metropolitan France

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/CitizenProject/CitizenProjectControllerSubscriber.php [hunk 1]
```diff
@@ -17,8 +17,7 @@
  */
 class CitizenProjectControllerSubscriber implements EventSubscriberInterface
 {
-    const DEPARTMENT_CODES = ['13', '16', '19', '29', '30', '34', '38', '47', '54', '56', '73', '78', '79', '80', '81', '83', '84', '87', '89', '91', '92'];
-    const POSTAL_CODES = ['75005', '75006', '75017', '75018'];
+    const UNAUTHORIZED_DEPARTMENT_CODES = ['971', '972', '973', '974', '975', '976', '984', '986', '987', '988'];
```
Comment (HeahDude) [inline src/CitizenProject/CitizenProjectControllerSubscriber.php]: Missing scope (i.e `public`).

File: src/CitizenProject/CitizenProjectControllerSubscriber.php [hunk 2]
```diff
@@ -17,8 +17,7 @@
  */
 class CitizenProjectControllerSubscriber implements EventSubscriberInterface
 {
-    const DEPARTMENT_CODES = ['13', '16', '19', '29', '30', '34', '38', '47', '54', '56', '73', '78', '79', '80', '81', '83', '84', '87', '89', '91', '92'];
-    const POSTAL_CODES = ['75005', '75006', '75017', '75018'];
```
Comment (HeahDude) [inline src/CitizenProject/CitizenProjectControllerSubscriber.php]: Are your sure those constants are not used?

Comment (illusionOfParadise) [reply src/CitizenProject/CitizenProjectControllerSubscriber.php]: Yes

File: src/CitizenProject/CitizenProjectControllerSubscriber.php (hunk 1 above)
Comment (illusionOfParadise) [reply src/CitizenProject/CitizenProjectControllerSubscriber.php]: Done

**Pull request merged**